articles/azure/azs-how-create-vm-terraform.md

```diff
@@ -8,14 +8,14 @@ toc_sub1: How To
 toc_sub2: Create a virtual machine
 toc_sub3:
 toc_sub4:
-toc_title: Create a virtual machine - Terraform
+toc_title: Create a VM using Terraform on Azure Stack
```

Comment:
This name change cause th artcle to ffall out of standard with the other "Create a virtual machine" articles